README.md

```diff
@@ -285,10 +275,10 @@ Let's go ahead and finish the tutorial on the on-premises instance.
     ```
 
     > For users in the mainland of China, please add the startup parameter：`--image-mirror-country=cn`.
-
-    ```bash
-    minikube start --image-mirror-country=cn
-    ```
+    >
+    > ```bash
+    > minikube start --image-mirror-country=cn
```

Comment:
`--image-mirror-country=cn` may not be enough
Specify the k8s version like 1.23.8 ?

ref:
- https://github.com/kubernetes/minikube/issues/15451#issuecomment-1409817778
- https://github.com/kubernetes/minikube/issues/15270#issuecomment-1304966090

Reply:
done